code/game/objects/items/storage/firstaid.dm

```diff
@@ -1,341 +1,353 @@
-
-/obj/item/storage/pill_bottle/bicaridine
-	name = "\improper Bicaridine pill bottle"
-	icon_state = "pill_canister11"
-	pill_type_to_fill = /obj/item/reagent_container/pill/bicaridine
-
-/obj/item/storage/pill_bottle/dexalin
-	name = "\improper Dexalin pill bottle"
-	icon_state = "pill_canister1"
-	pill_type_to_fill = /obj/item/reagent_container/pill/dexalin
-
-//Alkysine
-/obj/item/storage/pill_bottle/alkysine
-	name = "\improper Alkysine pill bottle"
-	icon_state = "pill_canister7"
-	pill_type_to_fill = /obj/item/reagent_container/pill/alkysine
-
-
-//imidazoline
-/obj/item/storage/pill_bottle/imidazoline
-	name = "\improper Imidazoline pill bottle"
-	icon_state = "pill_canister9"
-	pill_type_to_fill = /obj/item/reagent_container/pill/imidazoline
-
-//PERIDAXON
-/obj/item/storage/pill_bottle/peridaxon
-	name = "\improper Peridaxon pill bottle"
-	icon_state = "pill_canister10"
-	pill_type_to_fill = /obj/item/reagent_container/pill/peridaxon
-
-//RUSSIAN RED ANTI-RAD
-/obj/item/storage/pill_bottle/russianRed
-	name = "\improper Russian Red pill bottle"
-	icon_state = "pill_canister"
-	pill_type_to_fill = /obj/item/reagent_container/pill/russianRed
-
-
-/obj/item/storage/pill_bottle/quickclot
-	name = "\improper Quickclot pill bottle"
-	icon_state = "pill_canister8"
-	pill_type_to_fill = /obj/item/reagent_container/pill/quickclot
-
-
-//Ultrazine
-/obj/item/storage/pill_bottle/ultrazine
-	name = "\improper Pill bottle"
-	icon_state = "pill_canister11"
-	max_storage_space = 5
-	pill_type_to_fill = /obj/item/reagent_container/pill/ultrazine
-
-	req_access_txt = "200"
-	var/req_role = "Corporate Liaison"
-
-
-/obj/item/storage/pill_bottle/ultrazine/proc/id_check(mob/user)
-
-	var/mob/living/carbon/human/H = user
-
-	if(!allowed(user))
-		to_chat(user, "<span class='notice'>It must have some kind of ID lock...</span>")
-		return 0
-
-	var/obj/item/card/id/I = H.wear_id
-	if(!istype(I)) //not wearing an ID
-		to_chat(H, "<span class='notice'>It must have some kind of ID lock...</span>")
-		return 0
-
-	if(I.registered_name != H.real_name)
-		to_chat(H, "<span class='warning'>Wrong ID card owner detected.</span>")
-		return 0
-
-	if(req_role && I.rank != req_role)
-		to_chat(H, "<span class='notice'>It must have some kind of ID lock...</span>")
-		return 0
-
-	return 1
-
-/obj/item/storage/pill_bottle/ultrazine/attack_self(mob/living/user)
-	if(!id_check(user))
-		return
-	..()
-
-/obj/item/storage/pill_bottle/ultrazine/open(mob/user)
-	if(!id_check(user))
-		return
-	..()
+/* First aid storage
+ * Contains:
+ *		First Aid Kits
+ * 		Pill Bottles
+ */
+
+/*
+ * First Aid Kits
+ */
+/obj/item/storage/firstaid
+	name = "first-aid kit"
+	desc = "It's an emergency medical kit for those serious boo-boos."
+	icon_state = "firstaid"
+	throw_speed = 2
+	throw_range = 8
+	cant_hold = list("/obj/item/ammo_magazine",
+							"/obj/item/explosive/grenade",
+							"/obj/item/tool/surgery") //to prevent powergaming.

```

Comment:
This is a change that was made by someone else. Unless if you want to keep surgical tools out of first aid kits for some reason, remove it.